Userland/Libraries/LibSQL/AST/Expression.cpp

```diff
@@ -46,7 +42,7 @@ ResultOr<Value> ChainedExpression::evaluate(ExecutionContext& context) const
     for (auto& expression : expressions())
         values.unchecked_append(TRY(expression.evaluate(context)));
 
-    return Value { move(values) };
+    return TRY(Value::create_tuple(move(values)));
```

Comment:
No need for TRY,
```suggestion
    return Value::create_tuple(move(values));
```
should also work.